src/modules/Transcript/index.tsx

```diff
@@ -780,7 +793,22 @@ function useAudioSync(
         }
         // If we found a paragraph for the current audio, use it
         else if (audioParagraph) {
-            setCurrentParagraph(audioParagraph.id);
+            const scrollHeight = scrollContainer?.scrollHeight;
+            const offsetHeight = scrollContainer?.offsetHeight;
+            const scrollTop = scrollContainer?.scrollTop;
+            let isAtBottom = false;
+            if (scrollHeight && offsetHeight && scrollTop) {
+                // 180 gives us some buffer to decide they're close enough to the bottom
+                isAtBottom = scrollHeight <= 180 + offsetHeight + scrollTop;
+            }
+            if (isAtBottom && partial.text && !isListening) {
```

Comment:
If we have a selected paragraph, but the user has scrolled to the bottom, and we also have a partial, and we are not currently playing audio - we'll select the partial